docs/source/_templates/sdk_TP_tutorial.rst

```diff
@@ -109,53 +109,68 @@ and transaction family specific. When implementing a transaction handler the
 binary serialization protocol is up to the implementer.
 {% endif %}
 
-Without yet getting into the details of how this information is encoded, we can
-start to think about what ``apply`` needs to do. ``apply`` needs to
+To separate details of state encoding and payload handling from validation
+logic, the Xo example has XoState and XoPayload classes. The XoPayload has
```

Comment:
Cap and format nitpicks: Capitalize XO in "the XO example".  Also, please tag XoState and XoPayload as literals (surround with double back ticks).